### DISCUSSION TITLE
So is the user supposed to display a QR code on their phone and scan it with their desktop computer?

### DISCUSSION THREAD

**Discussion by zkhalapyan** [WalletConnect/walletconnect-monorepo#1229] [Developer Support]
There are not a lot of examples, but here is [one](https://github.com/skywinder/myWeb3Wallet/blob/b603943898f8d241edcfb74f6c7f96f766e61d03/myWeb3Wallet/ViewControllers/WalletController/WalletConnectV2/Proposer/ProposerViewController.swift#L68). The best I can understand this is that someone is display a QR code on their phone so that they can scan with their desktop?

**Comment by chadyj** [comment]
The most common mobile use case is using mobile linking https://docs.walletconnect.com/mobile-linking

However, WalletConnect also supports multi-devices, so you could show the QR on one device, and scan with another.
